react/utils/client/links/base64Link.ts

```diff
@@ -1,6 +1,10 @@
 import { ApolloLink, NextLink, Operation } from 'apollo-link'
 import { Base64 } from 'js-base64'
 
+import { OperationContext } from './uriSwitchLink'
+
+const MAX_QUERYSTRING_SIZE = 4096
```

Comment:
Is there some reasoning behind the value `4096`?

Reply:
No, it's just an empirical value